redaxo/src/addons/phpmailer/pages/config.php

```diff
@@ -32,7 +32,9 @@
         ['priority', 'int'],
         ['smtp_debug', 'int'],
         ['test_address', 'string'],
-        ['log', 'int', 1],
+        ['logging', 'int'],
+        ['log_archive', 'int'],
```

Comment:
Hier muss es `mail_archive` sein, oder?

Reply:
geändert in archive: https://github.com/redaxo/redaxo/pull/3109/commits/fc3234492ecf20200c183e02e6e8662fe0fd1167